bin/patch

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env perl
+# patch - Apply a patch to Webmin core or its modules from GitHub or a local file
+
+use strict;
+use warnings;
+
+use 5.010;
+
+use Getopt::Long qw(:config permute pass_through);
+use Pod::Usage;
+use Term::ANSIColor qw(:constants);
+use File::Basename;
+use Cwd qw(cwd);
+
+my %opt;
+GetOptions('help|h' => \$opt{'help'});
+pod2usage(0) if ($opt{'help'});
+
+# Get Webmin path
+my $path = cwd;
+my $lib = "web-lib-funcs.pl";
+if (!-r "$path/$lib") {
+	$path = dirname(dirname($0));
+	if (!-r "$path/$lib") {
+		$path = $path = Cwd::realpath('..');
+		}
+}
+
+# Check if curl is installed
+if (!`which curl`) {
```

Comment:
Can't we use has_command here?

Reply:
No, not really, as the env ins't set.

Reply:
What env? You mean $PATH? Because `which` also depends on that.